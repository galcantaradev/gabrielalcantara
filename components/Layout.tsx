import Head from 'next/head';
import { ReactNode } from 'react';

import NavBar from './NavBar';
import Footer from './Footer';

interface Props {
  children: ReactNode;
}

const Layout = (props: Props) => {
  return (
    <div className="min-h-screen min-w-screen bg-black-2 text-white text-lg flex flex-col">
      <Head>
        <title>Gabriel Alc√¢ntara</title>
      </Head>
      <NavBar />
      <div className="pt-10 flex-1 mt-12">{props.children}</div>
      <Footer />
    </div>
  );
};

export default Layout;
