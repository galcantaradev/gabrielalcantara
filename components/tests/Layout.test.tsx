import { render } from '@testing-library/react';

import { Layout } from '../Layout';

describe('Layout.test.tsx', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Layout>Children</Layout>);

    expect(asFragment).toMatchSnapshot();
  });
});
