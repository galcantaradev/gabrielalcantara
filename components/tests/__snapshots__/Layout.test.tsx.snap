// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout.test.tsx should render correctly 1`] = `[Function]`;
